### PR TITLE
Upgrade/documentation and readme

### PR DESCRIPTION
**Pull request by jdelamare**
This commit contains 100% English.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,43 @@
 # ereadingtool
-Build [![CircleCI](https://circleci.com/gh/ereadingtool/ereadingtool.svg?style=svg)](https://circleci.com/gh/ereadingtool/ereadingtool)
+The ereadingtool allows students and instructors to consume or provide a language's texts and interactively view its translations. Since languages rarely have a one-to-one mapping, texts created on the app can have their translation modified or changed. Students who consume the text material are presented questions created by an instructor for that particular text. Scores and progress is tracked and can be converted to a PDF for alternative viewing.
```
Comment (bgins) [inline README.md:2]: > Scores and progress is tracked...

Edit: is -> are

File: README.md [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+
+## Overview
+Ereadingtool is a SPA delivered by an NGINX webserver. The SPA communicates with a Django API server using a REST-ful paradigm. The Django web framework utilizes websocket connections for smooth interaction between the client and the server. It is supported by a Redis in-memory database to quickly serve the texts to the client over this websocket connection. Note that this is a microservice architecture.
```
Comment (bgins) [inline README.md:5]: Might be good to expand on "micoservice architecture".  Maybe something like "Note that this is a microservice architecture, but deployed to a single server."

File: README.md [hunk 3]
```diff
@@ -1,10 +1,43 @@
...
+The ereadingtool allows students and instructors to consume or provide a language's texts and interactively view its translations. Since languages rarely have a one-to-one mapping, texts created on the app can have their translation modified or changed. Students who consume the text material are presented questions created by an instructor for that particular text. Scores and progress is tracked and can be converted to a PDF for alternative viewing.
+
+## Overview
+Ereadingtool is a SPA delivered by an NGINX webserver. The SPA communicates with a Django API server using a REST-ful paradigm. The Django web framework utilizes websocket connections for smooth interaction between the client and the server. It is supported by a Redis in-memory database to quickly serve the texts to the client over this websocket connection. Note that this is a microservice architecture. 
+
+## Usage
+To run the app locally, follow the instructions in the **Web** section and the **Server** section. Either order is fine, just be sure to run the API server on a different port than the Elm pages are served from.
```
Comment (bgins) [inline README.md:8]: Edit: _Elm pages are_ -> _Elm app is_

File: README.md [hunk 4]
```diff
@@ -54,4 +87,35 @@ All debug statements must be removed for the optimized build.
...
+### Develop
+Be sure to have your virtualized environment activated. From the project's root directory run:
+
+```
+python manage.py runserver
+```
+
+## Build
+
+Build [![CircleCI](https://circleci.com/gh/ereadingtool/ereadingtool.svg?style=svg)](https://circleci.com/gh/ereadingtool/ereadingtool)
```
Comment (bgins) [inline README.md:113]: We might want to keep the badge up top. Nice to see the build status when you first see the README.

**Review by bgins — APPROVED** [review]
Nicely done! A few suggested edits for you to look at and this is good to go.

File: README.md (hunk 4 above)
Comment (jdelamare) [reply README.md:113]: Oh that's what that is! To the top it goes.

**Pull request merged**